trame_server/controller.py

```diff
@@ -69,7 +69,7 @@ def register_trigger(func):
             logger.info("trigger(%s)", name)
             self._triggers[name] = func
             self._triggers_fn2name[func] = name
-            return func
+            return func, name
```

Comment:
I think this is going to break the decorator usage. 

```python
@ctrl.trigger("hello")
def say_hello():
   print("hello")
```

Reply:
yeah I was afraid of that.. guess we might want not to apply the namespace then. see my latest comment